src/analysis/special_functions/gamma.lean

```diff
@@ -519,3 +534,425 @@ end
+        ring,  },
+      { rw [nat.succ_eq_add_one, nat.cast_add, nat.cast_one, neg_add] at hs',
+        linarith }  },
+    rw [Gamma_add_one, mul_ne_zero_iff] at this,
+    { exact this.2 },
+    { simpa using hs 0 } },
+end
+
+lemma differentiable_at_Gamma {s : ℝ} (hs : ∀ m:ℕ, s + m ≠ 0) : differentiable_at ℝ Gamma s :=
+begin
+  apply has_deriv_at.differentiable_at,
+  apply has_deriv_at.real_of_complex,
+  apply differentiable_at.has_deriv_at,
+  apply complex.differentiable_at_Gamma,
+  simp_rw [←complex.of_real_nat_cast, ←complex.of_real_add, complex.of_real_ne_zero],
+  exact hs,
+end
+
+lemma Gamma_mul_add_mul_le_rpow_Gamma_mul_rpow_Gamma {s t p : ℝ}
+  (hs : 0 < s) (ht : 0 < t) (hp : 0 < p) (hP : p < 1) :
+  Gamma (p * s + (1 - p) * t) ≤ Gamma s ^ p * Gamma t ^ (1 - p) :=
+begin
+  rw [Gamma_eq_integral hs, Gamma_eq_integral ht, Gamma_eq_integral],
+  swap, { exact add_pos_of_pos_of_nonneg (mul_pos hp hs) (mul_nonneg (by linarith) ht.le) },
+  -- will apply Hoelder to `f p s` and `f (1 - p) t`:
+  let f : ℝ → ℝ → ℝ → ℝ := λ q u x, exp (-q * x) * x ^ (q * (u - 1)),
+  -- some properties of f:
+  have posf : ∀ (q u x : ℝ), x ∈ Ioi (0:ℝ) → 0 ≤ f q u x :=
+    λ q u x hx, mul_nonneg (exp_pos _).le (rpow_pos_of_pos hx _).le,
+  have posf' : ∀ (q u : ℝ), ∀ᵐ (x : ℝ)  ∂volume.restrict (Ioi 0), 0 ≤ f q u x :=
+    λ q u, (ae_restrict_iff' measurable_set_Ioi).mpr (ae_of_all _ (posf q u)),
+  have fpow : ∀ {x : ℝ}, x ∈ Ioi (0:ℝ)  → ∀ {q : ℝ}, (0 < q) →
+    ∀ (u : ℝ), exp (-x) * x ^ (u - 1) = f q u x ^ (1 / q),
+  { intros x hx q hq u,
+    dsimp only [f],
+    rw [mul_rpow (exp_pos _).le ((rpow_nonneg_of_nonneg $ le_of_lt hx) _), ←exp_mul,
+      ←rpow_mul (le_of_lt hx)],
+    congr' 2;
+      { field_simp [hq.ne'], ring } },
+  -- key property `f q u` is in `ℒq`:
+  have f_mem_Lq : ∀ {q : ℝ}, (0 < q) → ∀ {u : ℝ}, (0 < u) →
+    mem_ℒp (f q u) (ennreal.of_real (1 / q)) (volume.restrict (Ioi 0)),
+  { intros q hq u hu,
+    rw [←mem_ℒp_norm_rpow_iff, ennreal.to_real_of_real (one_div_nonneg.mpr hq.le), ennreal.div_self,
+      mem_ℒp_one_iff_integrable],
+    all_goals { try { rwa [ne.def, ennreal.of_real_eq_zero, not_le, one_div_pos] } },
+    all_goals { try { exact ennreal.of_real_ne_top } },
+    { apply integrable.congr (Gamma_integral_convergent hu),
+      refine eventually_eq_of_mem (self_mem_ae_restrict measurable_set_Ioi) (λ x hx, _),
+      dsimp only,
+      rw fpow hx hq,
+      congr' 1,
+      exact (norm_of_nonneg (posf _ _ x hx)).symm },
+    { refine continuous_on.ae_strongly_measurable _ measurable_set_Ioi,
+      refine (continuous.continuous_on _).mul (continuous_at.continuous_on (λ x hx, _)),
+      { exact continuous_exp.comp (continuous_const.mul continuous_id'), },
+      { exact continuous_at_rpow_const _ _ (or.inl (ne_of_lt hx).symm), } } },
+  have e : (1 / p).is_conjugate_exponent (1 / (1 - p)) := ⟨by rwa [lt_div_iff hp, one_mul],
+    by rw [one_div_one_div, one_div_one_div, ←add_sub_assoc, add_sub_cancel']⟩,
```

Comment:
This is maybe worth registering as a general lemma in the `is_conjugate exponent` file.